src/redux/home/tests/home.sagas.test.js

```diff
@@ -0,0 +1,76 @@
+import { expectSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+import {
+  getHomePageLooksImages,
+  updateHomePageLooksImage
+} from '../home.operations';
+
+import {
+  handleHomePageImagesUpdate,
+  handleHomePageImagesLoad,
+  handleHomePageError
+} from '../home.sagas';
+
+import {
+  setHomePageData,
+  setHomePageDataLoading,
+  updateHomePageImagesInStore,
+  setHomePageDataError
+} from '../home.actions';
+
+import {
+  looksImageUpdateMock,
+  looksImagesMock,
+  updatedImages,
+  id as _id,
```

Comment:
Try not to change the name of the imported variable because it`s harder to follow in code

Reply:
Done